docs/cli/discover_and_take.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+CLI_PATH=$1
+YOUR_MONERO_ADDR=$2
+
+RENDEZVOUS_ADDR="/dnsaddr/rendezvous.coblox.tech/p2p/12D3KooWQUt9DkNZxEn2R5ymJzWj15MpG6mTW84kyd8vDaRZi46o"
```

Comment:
This is defaulted no? Why are you specifying it again in the script? Just to showcase that it is possible?

Reply:
thanks - I just forgot the defaults. I redoced it to just the address, but it can be removed completely :)